src/components/tables/LMTable/LMTable.vue

```diff
@@ -215,27 +186,25 @@ export default defineComponent({
     }
 
     function calculatePricesFor(totals: TokenTotal[]) {
-      let totalUsd = 0;
+      let totalFiat = 0;
       for (const total of totals) {
-        const usdValue =
-          prices.value[total.token.toLowerCase()].price * total.total;
-        totalUsd = totalUsd + usdValue;
+        const usdValue = priceFor(getAddress(total.token)) * total.total;
```

Comment:
Sorry to be picky but could we call this `fiatValue`
```suggestion
        const fiatValue = priceFor(getAddress(total.token)) * total.total;
```